conf/env/PRESTASHOP.env

```diff
@@ -16,6 +16,7 @@ PS_FOLDER_INSTALL=installOLD
 ADMIN_MAIL=demo@reverb.com
 ADMIN_PASSWD=123456
 PS_HANDLE_DYNAMIC_DOMAIN=0
+PS_ERASE_DB=1
```

Comment:
By default Yes, if you don't want re-install the database, you can edit the PRESTASHOP.env : PS_ERASE_DB=0.